frontend/apps/filemanager/filemanager.lua

```diff
@@ -316,7 +320,12 @@ function FileManager:init()
                         }
                         UIManager:show(script_is_running_msg)
                         UIManager:scheduleIn(0.5, function()
-                            local rv = os.execute(BaseUtil.realpath(file))
+                            local rv = 0;
+                            if Device:isAndroid() then
+                                  rv = os.execute("sh " .. BaseUtil.realpath(file)) -- run by sh, because sdcard has no execute permissions
```

Comment:
```suggestion
                                rv = os.execute("sh " .. BaseUtil.realpath(file)) -- run by sh, because sdcard has no execute permissions
```